cmd/istio-router-check/README.md

```diff
@@ -0,0 +1,15 @@
+# Istio Router Check
```

Comment:
Would be good to link to this from the `README.md` at the root of the repository

Reply:
I'd prefer to keep it here and not advertise too much until is a bit more mature/tested internally.